kustomize/commands/internal/testutils/remove_testutils.go

```diff
@@ -41,16 +40,20 @@ type Case struct {
 	Expected Expected
 }
```

Comment:
So, Maybe those global struct names are a little generic more than real functions.
If you can, Could you rename those to a more small scope name? (ex, add the `Remove` prefix, It's only my idea.)

Reply:
Sure! Maybe `RemoveTestCase`?

Reply:
@koba1t updated! Could you PTAL?

Reply:
Thanks!